### ISSUE TITLE
Outlook 2010: Focus not announced when Page Up/Down used in Address Book

### ISSUE THREAD

**Issue by elliott94** [nvaccess/nvda#5721]
STR:
1. Open Outlook's Address Book, and navigate to the list that shows all currently stored names/addresses.
2. Press the Page Down key.

Expected: NVDA should announce the name of the user that now has focus.

Actual: Despite the focus moving down the list, NVDA doesn't announce anything until a cursor key is pressed which shows that the focus was previously moved.

**Comment by jcsteh** [comment]
Any thoughts @MichaelDCurran? Do we fake focus ourselves for the arrows 
here?

**Comment by Adriani90** [comment]
This issue is still reproducible.

**Comment by Adriani90** [comment]
This is not hapenning anymore with NVDA 2023.1 in Office 2016 or newer. Given the support of older Office versions ended completely, I am closing this as abandoned since I don't expect any further updates to to this Office version. If you are still having this issue, please open a new issue for office versions newer than 2013.
